doc-content/enterprise-only/springboot/bus-app-swagger_proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='bus-app-swagger_{context}']
+= Enable Swagger documentation
+
+You can enable Swagger-based documentation for all endpoints
+available in the service project of your Red Hat Business Automation business application.
+
+.Prerequisite
+You have a `<business-application>.zip` file that you created using the http://start.jbpm.org[business applications] web site.
+
+. Unzip the `<business-application>.zip` file and navigate to the `<business-application>/<business-application>-service` folder.
+. Open the service project `pom.xml` file in a text editor.
+. Add the following dependencies to service project `pom.xml` file and save the file.
++   
+[source, xml]
+----
+<dependency>
+  <groupId>org.apache.cxf</groupId>
+  <artifactId>cxf-rt-rs-service-description-swagger</artifactId>
+  <version>3.1.11</version>
+</dependency>
+<dependency>
+  <groupId>io.swagger</groupId>
+  <artifactId>swagger-jaxrs</artifactId>
+  <version>1.5.15</version>
+  <exclusions>
+    <exclusion>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>jsr311-api</artifactId>
+    </exclusion>
+  </exclusions>
+</dependency>
+<dependency>
+  <groupId>org.webjars</groupId>
+  <artifactId>swagger-ui</artifactId>
+  <version>2.2.10</version>
+</dependency>
```

Comment:
I would do it the same as in community docs - I would write that this is only for Swagger user interface, in theory it is not mandatory. So it would be good to have it as close to community as possible.